README.md

```diff
@@ -1,4 +1,4 @@
-<p align="right">🌎 <a href="https://github.com/redaxo/redaxo/blob/master/README.de_DE.md">Deutsch</a></p>
+<p align="right">🌎 <a href="https://github.com/redaxo/redaxo/blob/master/README.de.md">Deutsch</a></p>
```

Comment:
@schuer fyi, wir werden vermutlich die datei hier umbennen, da es unwahrscheinlich ist, dass wir hier in der gleichen sprache je nach land unterschiedliche files haben werden

Reply:
Finde ich prima.